src/components/reusables/CompactDeletableListItem/CompactDeletableListItem.tsx

```diff
@@ -55,21 +58,24 @@ export const CompactDeletableListItem = ({
             <div className={cn(classes.leftText, classes[`leftText__${contentColor}`])}>
               {leftText}
             </div>
-            <div className={(classes.middleText, classes[`middleText__${contentColor}`])}>
-              {middleText}
-            </div>
-          </div>
-          <div className={classes.deleteSection}>
-            {removeCallback && (
-              <Button
-                variant={ButtonVariant.Quiet}
-                color={ButtonColor.Danger}
-                icon={<MinusCircle />}
-                onClick={removeCallback}
-              >
-                {t('api_delegation.delete')}
-              </Button>
+            {!isPhoneScreen && (
```

Comment:
Why isPhoneScreen here when isXs was used earlier? Should probably stay consistent